greenpeace/planet4-master-theme
# PLANET-7489: Migrate Campaign Covers Blocks to Columns blocks

Ref: https://jira.greenpeace.org/browse/PLANET-7489

<hr>

**DESCRIPTION:**
This PR migrates the Campaign Covers blocks to Columns blocks.

<hr>

**TESTING INSTRUCTIONS:**
1. Run this PR in your local environment.
2. Identify a post using a Campaign content block.
3. Run the migrator command: `npx wp-env run cli wp p4-run-activator`
4. Check whether the migration worked as expected.
5. To reset the environment database back to the latest default content and repeat the tests, run: `npm run db:reset`.
